man-db.yaml

```diff
@@ -71,6 +71,11 @@ test:
       packages:
         - man-db-doc
   pipeline:
+    - name: Ensure that setuid bits are not set
+      runs: |
+        set -x
+        if [ -u /usr/bin/man ]; then exit 1; fi
+        if [ -u /bin/man ]; then exit 1; fi
```

Comment:
Also, please double check the `/usr/bin/mandb` binary:

```
2a4cfbc6f0f4:/work/packages# ls -la /usr/bin/ | grep -v -E '^(l|d|-rwx)'
total 7536
-rwsr-sr-x    1 man      man         148792 Mar 31 16:07 man
-rwsr-sr-x    1 man      man         176712 Mar 31 16:07 mandb
```

Reply:
Added in [`dce3443` (#52093)](https://github.com/wolfi-dev/os/pull/52093/commits/dce3443e5501643c3f15c76dc91def12e6902048).